Comments/ClientApp/src/components/Document/__tests__/__snapshots__/Document.test.js.snap

```diff
@@ -6,906 +6,931 @@ Array [
   "",
   <aside
     className="Tutorial mt--0 pt--d pb--d"
->
+  >
```

Comment:
any idea why the formatting of this changed?

it makes it really difficult/impossible to see whether the output's different/broken.